frame/dvm/src/tests.rs

```diff
@@ -173,59 +173,70 @@ fn get_wkton_balance(sender: &AccountInfo, nonce: u64) -> U256 {
 }
 
 fn wkton_balance_input(address: H160) -> Vec<u8> {
+	#[allow(deprecated)]
 	let func = Function {
 		name: "balanceOf".to_owned(),
 		inputs: vec![Param {
 			name: "address".to_owned(),
 			kind: ParamType::Address,
+			internal_type: Some("address".into()),
 		}],
 		outputs: vec![],
 		constant: false,
+		state_mutability: StateMutability::NonPayable,
```

Comment:
why the old version this `constant` is false?
the balanceOf function in wkton is a view function, here constant should be `true`, and `state_mutability=StateMutability::View`

Reply:
I think you are right! Update this  BTW? @AurevoirXavier

Reply:
Not sure. Recommend you to take a full review after this.